pandas/tests/groupby/test_groupby.py

```diff
@@ -27,6 +27,31 @@ def test_repr():
     assert result == expected
 
 
+def test_groupby_std():
```

Comment:
these can go in test_function.py near the other tests for std.
add a 1-liner on what you are testing exactly here

Reply:
Moved thanks